src/ucp/api/ucp.h

```diff
@@ -863,10 +863,10 @@ ucs_status_t ucp_rmem_ptr(ucp_ep_h ep, void *remote_addr, ucp_rkey_h rkey,
  *                          that the call-back is only invoked in a case when
  *                          the operation cannot be completed in place.
  *
- * @return UCS_OK           - The send operation was completed completed
- *                          immediately.
+ * @return UCS_OK           - The send operation was completed immediately.
  * @return UCS_PTR_IS_ERR(_ptr) - The send operation failed.
- * @return otherwise        - Operation was scheduled for send. The request handle
+ * @return otherwise        - Operation was scheduled for send or was already
+ *                          completed (after scheduling). The request handle
```

Comment:
@shamisp the user may assume that since UCS_OK wasn't returned, the operation wasn't completed. (because if it was completed then UCS_OK would be returned). the user is not aware of the fact that we do two send attempts inside this function.

we received a request to make this more clear from a user who misused this function and said that the explanation wasn't clear enough so we decided to just elaborate a bit more to make it clearer.